radar.csv

```diff
@@ -80,10 +52,7 @@ Babel,adopt,languages and frameworks,false,"Babel is a JavaScript compiler allow
 "XML",endure,tools,false,"For storing user information we used to use XML. We'll continue to use that, but avoid it in the future because it's very verbose. We perceive the cost of migration away to be high enough for this to be Endure, rather than Retire."
 "Registry",endure,tools,false,"We avoid storing settings in the Registry as it's Windows specific. However, it does make it easier for system administrators to apply group policy so we should bear this in mind when choosing setting locations."
 "Feature flags",adopt,techniques,false,"<a href='https://martinfowler.com/articles/feature-toggles.html'>A mechanism to prevent code in master from affecting the way the product behaves</a>. There are lots of different types of flags/toggles: they might be set at compile or run time; they might be purely internal or visible to the user, and so on."
-"ReSpeller using AmE",adopt,tools,false,"<a href='https://www.jetbrains.com/help/resharper/Spell_Checking.html'>ReSpeller</a> is a spell-checking engine integrated with ReSharper. We use American English to be consistent with .NET Framework and other 3rd party libraries. Additional words are stored in a solution team-shared settings file (*.sln.DotSettings)."
-"Open API specification",adopt,techniques,false,"Our public web-based APIs are exposed through the <a href='https://www.openapis.org/'>Open API</a> specification interface as being the description for REST APIs, which allows both humans and computers to discover and understand the capabilities of a service without requiring access to source code, additional documentation, or inspection of network traffic."
```

Comment:
This is the only one I'm not sure about - We have the API policy that says everything should be documented - but we don't specify how. Should we have this on the radar as our preferred approach until it becomes policy?

Reply:
I think the specification isn't an adoptable thing, we'd need to be more explicit about what to use.

Swagger is being explored by the Platform team, so I'll add that to Libraries?